src/components/help/Feedback.js

```diff
@@ -0,0 +1,186 @@
+                                    </Button>
+                                </>
+                            }
+                        >
+                            {error && (
+                                <ErrorTypographyWithDialog
+                                    errorMessage={i18nHelp("feedback_error")}
+                                    errorObject={error}
+                                />
+                            )}
+                            <Typography>
+                                {i18nHelp("feedback_welcome")}
+                            </Typography>
+
+                            {feedbackStatus === constants.LOADING && (
+                                <CircularProgress
+                                    size={30}
+                                    thickness={5}
+                                    style={{
+                                        position: "absolute",
+                                        top: "50%",
+                                        left: "50%",
+                                    }}
+                                />
+                            )}
+                            {!userProfile?.id && (
+                                <>
+                                    <Field
+                                        id={build(baseId, ids.NAME)}
+                                        name={"name"}
+                                        required={true}
+                                        component={FormTextField}
+                                        validate={(value) =>
+                                            nonEmptyField(value, t)
+                                        }
+                                        label={i18nHelp("name")}
+                                    />
+                                    <Field
+                                        id={build(baseId, ids.EMAIL)}
+                                        name="email"
+                                        required={false}
+                                        component={FormTextField}
+                                        label={i18nHelp("email")}
+                                        validate={validateEmail}
+                                        helperText={i18nHelp("feedback_helper")}
+                                    />
+                                </>
+                            )}
+                            <Field
+                                id={build(baseId, ids.FEEDBACK)}
+                                name="feedback"
+                                label={i18nHelp("feedback_title")}
+                                required={true}
+                                onKeyDown={(event) => {
+                                    if (event.key === "Enter") {
+                                        handleSubmit();
+                                    }
+                                }}
```

Comment:
I don't think we want this key handler for a `FormMultilineTextField`.

The point of that field is to allow the user to enter multiple lines of text, separated by newlines, but I think this handler will submit the form when they try to start a new line in this field.